python/feathub/processors/flink/table_builder/flink_table_builder.py

```diff
@@ -155,7 +155,7 @@ def build(
     def _filter_table_by_keys(
         self,
         table: NativeFlinkTable,
-        keys: Union[pd.DataFrame, TableDescriptor],
+        keys: Union[pd.DataFrame, TableDescriptor, None],
```

Comment:
This method should only be called when keys != None. Why do we need this change?